index.bs

```diff
@@ -1935,25 +1930,41 @@ partial interface MLGraphBuilder {
+                    1. Let |filterInputChannels| be |filterShape|[2].
+                    1. Let |outputChannels| be |filterShape|[3].
                 : {{MLConv2dFilterOperandLayout/"ohwi"}}
+                ::
+                    1. Let |outputChannels| be |filterShape|[0].
+                    1. Let |filterHeight| be |filterShape|[1].
+                    1. Let |filterWidth| be |filterShape|[2].
+                    1. Let |filterInputChannels| be |filterShape|[3].
                 : {{MLConv2dFilterOperandLayout/"ihwo"}}
                 ::
+                    1. Let |filterInputChannels| be |filterShape|[0].
                     1. Let |filterHeight| be |filterShape|[1].
                     1. Let |filterWidth| be |filterShape|[2].
+                    1. Let |outputChannels| be |filterShape|[3].
                 : {{MLConv2dFilterOperandLayout/"oihw"}}
                 ::
+                    1. Let |outputChannels| be |filterShape|[0].
+                    1. Let |filterInputChannels| be |filterShape|[1].
                     1. Let |filterHeight| be |filterShape|[2].
                     1. Let |filterWidth| be |filterShape|[3].
             </dl>
         1. Let |outputSizes| be the result of [=MLGraphBuilder/calculating conv2d output sizes=] given |inputHeight|, |inputWidth|, |filterHeight|, |filterWidth|, |options|.{{MLConv2dOptions/padding}}, |options|.{{MLConv2dOptions/strides}}, and |options|.{{MLConv2dOptions/dilations}}.
         1. Switch on |options|.{{MLConv2dOptions/inputLayout}}:
             <dl class=switch>
                 : {{MLInputOperandLayout/"nchw"}}
-                :: Let |outputShape| be « |batches|, |channels|, floor( |outputSizes|[0] ), floor( |outputSizes|[1] ) ».
+                :: Let |outputShape| be « |batches|, |outputChannels|, floor( |outputSizes|[0] ), floor( |outputSizes|[1] ) ».
                 : {{MLInputOperandLayout/"nhwc"}}
-                :: Let |outputShape| be « |batches|, floor( |outputSizes|[0] ), floor( |outputSizes|[1] ), |channels|  ».
+                :: Let |outputShape| be « |batches|, floor( |outputSizes|[0] ), floor( |outputSizes|[1] ), |outputChannels|  ».
             </dl>
-    1. If |outputShape| is not the same as the shape of |options|.{{MLConv2dOptions/bias}}'s [=MLOperand/shape=], then [=exception/throw=] a {{TypeError}}.
+    1. If |inputChannels| % |options|.{{MLConv2dOptions/groups}} is not 0, then [=exception/throw=] a {{TypeError}}.
+    1. Else if |inputChannels| / |options|.{{MLConv2dOptions/groups}} is not equal to |filterInputChannels|, then [=exception/throw=] a {{TypeError}}.
+    1. If |options|.{{MLConv2dOptions/bias}} [=map/exists=]:
+        1. If |options|.{{MLConv2dOptions/bias}}'s [=MLOperand/rank=] is not 1, then [=exception/throw=] a {{TypeError}}.
+        1. If |outputChannels| is not equal to |options|.{{MLConv2dOptions/bias}}'s [=MLOperand/shape=][0], then [=exception/throw=] a {{TypeError}}.
```

Comment:
I know this is just a move/indent of the previous text, but i'd switch the order of the comparison so it matches the previous and next lines, i.e. start with _"If options.bias's ..."_

Reply:
fixed in https://github.com/webmachinelearning/webnn/pull/608/commits/3b11a3a6589703a91ab18e259c2468a524acd802